PowerShellTools/DebugEngine/ScriptDebugger.cs

```diff
@@ -464,7 +468,16 @@ internal void OpenRemoteFile(string fullName)
                 }
                 catch (Exception ex)
                 {
-                    Log.Error("Failed to open remote file through powershell remote session", ex);
+                    String err;
+                    if (forRemote)

```

Comment:
I don't feel quite useful about this flag, still in order to determine if it is under local/remotesession/remoteprocess, we need find out a better way instead of having those flags flow around.